generators/environment/templates/bin/pre-install.sh

```diff
@@ -10,11 +10,6 @@
 # of `grunt install`.
 ##
 
-if [ ! -f src/sites/default/default.settings.php ]; then
-  echo "Please run pre-install after successfully building the docroot."
-  exit 1
-fi
```

Comment:
The conditional below is specifically set up so an existing settings.php is not clobbered. The initial check is a more general "does Drupal codebase exist, regardless of state" check, so that no other pre-install operations are unnecessarily fired, since without that the install can be assumed to be going to crash immediately anyway.